### DISCUSSION TITLE
Server Error

### DISCUSSION THREAD

**Discussion by muhammedctgr** [prisma/prisma#15459] [Q&A]
### Bug description

I am trying to activate the next-auth with my github. Everything is configured fine, when I connected it with local host and configured the oauth app on github, i can use my github to login to the app, but when I try to connect the nextauth_url with the url of my prospective vercel app...its saying server error.
![Screenshot 2022-09-20 184435](https://user-images.githubusercontent.com/73699892/191450139-a31d2cf4-82a9-4a5b-9d67-70e9c079b885.jpg)

### Expected behavior

you should be able to login with your github account

### Prisma information

```prisma
model Post {
  id        String  @id @default(cuid())
  title     String
  content   String?
  published Boolean @default(false)
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  String?
}

model Account {
  id                 String  @id @default(cuid())
  userId             String  @map("user_id")
  type               String
  provider           String
  providerAccountId  String  @map("provider_account_id")
  refresh_token      String?
  access_token       String?
  expires_at         Int?
  token_type         String?
  scope              String?
  id_token           String?
  session_state      String?
  oauth_token_secret String?
  oauth_token        String?

  user User @relation(fields: [userId], references: [id], onDelete: Cascade)

  @@unique([provider, providerAccountId])
}

model Session {
  id           String   @id @default(cuid())
  sessionToken String   @unique @map("session_token")
  userId       String   @map("user_id")
  expires      DateTime
  user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
}

model User {
  id            String    @id @default(cuid())
  name          String?
  email         String?   @unique
  emailVerified DateTime?
  image         String?
  posts         Post[]
  accounts      Account[]
  sessions      Session[]
}

model VerificationToken {
  id         Int      @id @default(autoincrement())
  identifier String
  token      String   @unique
  expires    DateTime

  @@unique([identifier, token])
}


```

```ts
import { PrismaClient } from '@prisma/client';

let prisma: PrismaClient;

if (process.env.NODE_ENV === 'production') {
  prisma = new PrismaClient();
} else {
  if (!global.prisma) {
    global.prisma = new PrismaClient();
  }
  prisma = global.prisma;
}

export default prisma;
```

### Environment & setup

- OS: Windows
- Database: Heroku Postgres
- Node.js version: Node v16.13.2

### Prisma Version

```

```

**Comment by muhammedctgr** [comment]
visit https://p-hammed-blogr-nextjs-prisma.vercel.app/ to try and login

**Comment by nurul3101** [comment]
Hey @muhammedctgr 👋 
It's difficult to determine what the error could be without looking at the application code. Do you get any errors logged in the browser console or in your application server?

**Reply by vladi-stevanovic** [reply]
👋 Hello @muhammedctgr - did you perhaps have a chance to check if you get any errors logged in the browser console or in your application server? Let us know if this is still an open issue or we'd love to know how you resolved it!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
